src/app/pages/RewardPage/components/FeesEarnedTab/index.tsx

```diff
@@ -66,7 +77,29 @@ export const FeesEarnedTab: React.FC<IFeesEarnedTabProps> = ({
                       />
                     </div>
                   </th>
-                  <th></th>
+                  <th>
+                    <ActionButton
+                      text={t(translations.rewardPage.claimForm.ctaAll)}
+                      onClick={claim}
+                      className={classNames(
+                        'tw-border-none tw-px-4 xl:tw-px-2 2xl:tw-px-4',
+                        {
+                          'tw-cursor-not-allowed': isClaimDisabled,
+                        },
+                      )}
+                      textClassName="tw-text-xs tw-overflow-visible tw-font-bold"
+                      disabled={isClaimDisabled}
+                      title={
+                        (claimFeesEarnedLocked &&
+                          t(translations.maintenance.claimRewards).replace(
+                            /<\/?\d+>/g,
```

Comment:
Why do you need this? I think you should provide Sovryn Discord URL instead.

Reply:
This was already there, I just moved moved that button to another place. Will double check what it tries to do here

Reply:
Ok, this is title which can't have html - made a new translation key for this.